.env.local

```
@@ -1 +1,7 @@
 OPENPASS_API_KEY=your_key_here
+OPENAI_API_KEY=sk-...
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OPENAI_API_KEY key should go before the OPENPASS_API_KEY key

(UnorderedKey)